packages/styles/src/themes/pentahoPlus.ts

```diff
@@ -390,41 +390,114 @@ const pentahoPlus = makeTheme((theme) => ({
     HvButton: {
       classes: {
         root: {
-          borderRadius: theme.radii.full,
+          border: "none",
+          borderRadius: 48,
           padding: theme.spacing(0, "sm"),
-          "&:active": {
-            boxShadow: `inset 0 1px 2px 0 #00000033`, // 33 = 20% opacity
+          [`&[data-color="positive"]`]: {
+            "&:hover": {
+              backgroundColor: theme.palette.green[700],
```

Comment:
I think we should be using the colors directly, like elsewhere. These probably give a false sense old security (eg. they come from `theme.`, so they must be themmable) but they're not. It's just `green[700]` with extra steps.
I believe this would also be a blocker if we want to make the theme dynamic (not sure yet)

```suggestion
              backgroundColor: green[700],
```